src/pages/Dashboard/components/PeopleListItem.jsx

```diff
@@ -1,14 +1,25 @@
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+
 function PeopleListItem(props) {
```

Comment:
you can probably merge this with the PeopleList component since the code is very short